packages/icon/cli/clean-svgs.js

```diff
@@ -2,17 +2,25 @@ const { promises: fs } = require('fs')
 const fg = require('fast-glob')
 const path = require('path')
 const SVGO = require('svgo')
+
 const svgo = new SVGO({
   plugins: [
     { removeXMLNS: true },
     { removeDimensions: true },
     { removeAttrs: { attrs: '(stroke|fill)' } }
   ]
 })
-const regex = /aria-label=(?:"|')([\w ]+)(?:"|')/
-const addAria = (svg, name) =>
-  regex.test(svg) ? svg : svg.replace(/<svg/, `<svg aria-label="${name} icon"`)
+
+const ariaLabelRegEx = /aria-label=(?:"|')([\w ]+)(?:"|')/
```

Comment:
I don't remember this change last time.  But this time, I'm not seeing any change that seems significant.  Am I missing something?

Reply:
nothing significant. just trying to name the vars